app/src/main/java/com/poupa/vinylmusicplayer/dialogs/BlacklistFolderChooserDialog.java

```diff
@@ -50,24 +54,27 @@ private String[] getContentsArray() {
             results[0] = "..";
         }
         for (int i = 0; i < parentContents.length; i++) {
-            results[canGoUp ? i + 1 : i] = parentContents[i].getName();
+            results[canGoUp ? i + 1 : i] = isAtRootLevel() ? parentContents[i].getAbsolutePath() : parentContents[i].getName();
```

Comment:
When you are at the root level, it makes more sense to show the absolute path, since some items might be the complete path for a storage option (ie. `/storage/emulated/0`.)